app/code/core/Mage/GoogleAnalytics/Block/Ga.php

```diff
@@ -380,9 +380,9 @@ protected function _getOrdersTrackingCodeAnalytics4()
             $result[] = "gtag('event', 'view_item_list', " . json_encode($eventData, JSON_THROW_ON_ERROR) . ");";
         }
 
-        /** 
+        /**
          * This event signifies that a user viewed their cart.
-         * 
+         *
```

Comment:
A user viewed his cart.

Reply:
done

Reply:
this hints are copied from analytics docs